apps/search/scripts/migrations/recreate-webhooks.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable turbo/no-undeclared-env-vars */
+
+import { createGraphQLClient } from "@saleor/apps-shared";
+import { WebhookActivityTogglerService } from "../../src/domain/WebhookActivityToggler.service";
+import { FetchOwnWebhooksDocument } from "../../generated/graphql";
+import { AuthData } from "@saleor/app-sdk/APL";
+
+export const recreateWebhooks = async ({
+  authData,
+  dryRun,
+}: {
+  authData: AuthData;
+  dryRun: boolean;
+}) => {
+  console.log("Working on env: ", authData.saleorApiUrl);
+
+  const client = createGraphQLClient({
+    saleorApiUrl: authData.saleorApiUrl,
+    token: authData.token,
+  });
+  const webhooks = await client
+    .query(FetchOwnWebhooksDocument, {
+      id: authData.appId,
+    })
+    .toPromise()
+    .then((r) => r.data?.app?.webhooks);
+
+  if (!webhooks?.length) {
+    console.error("The environment does not have any webhooks, skipping");
+    return;
+  }
+
+  // Use currently existing webhook data to determine a proper baseUrl and enabled state
+  const targetUrl = webhooks[0].targetUrl;
```

Comment:
what if there is more than 1 webhook?

Reply:
I'm just using 2 things shared among all of the hooks - base URL and enabled status. The application always have the same values across all of the hooks.